import-dashboards.py

```diff
@@ -529,9 +530,84 @@ def set_home_dashboard(api_key):
     #             "SELECT 1, 1, 0, 0, id, '', '', datetime('now'), datetime('now') from dashboard WHERE slug='home'")
 
 
+def dbaas_dashboard(api_key):
```

Comment:
Can't we provision dashboards via a native Grafana provisioning mechanism as explained here https://grafana.com/tutorials/provision-dashboards-and-data-sources/#5?

Reply:
We investigated possibility of using provisioning mechanism some time ago. 
But it was refused to use due to that fact that our dashboards are a part of pmm-app plugin.
Also this mechanism is preventing possibility of modifying dashboards by users.